app/src/components/templateEditor/language/fStringTemplating/fStringTemplating.syntax.grammar

```diff
@@ -0,0 +1,25 @@
+@top FStringTemplate {(Template | Text | LEscape | REscape)*}
+
+@skip {} {
+  Template { LBrace Variable+ RBrace }
+}
+
+@local tokens {
+  RBrace { "}" }
```

Comment:
not sure if you can write comments in here but might be nice to have some since this is pretty foreign. 

Why is Rbrace in @local tokens and LBrace in @tokens?

Reply:
This will be difficult information to capture within comments im afraid to say 😂 

I can link to relevant sections in the lezer docs if that helps

Reply:
yeah maybe just linking to some docs at the top of these files would be nice!

Reply:
Done